ccdproc/ccddata.py

```diff
@@ -505,19 +516,20 @@ def fits_ccddata_reader(filename, hdu=0, unit=None, hdu_uncertainty='UNCERT',
 
     Parameters
     ----------
-
     filename : str
         Name of fits file.
 
     hdu : int, optional
-        FITS extension from which CCDData should be initialized.  If zero and
+        FITS extension from which CCDData should be initialized. If zero and
         and no data in the primary extention, it will search for the first
-        extension with data.  The header will be added to the primary header.
+        extension with data. The header will be added to the primary header.
+        Default is ``0``.
 
-    unit : astropy.units.Unit, optional
+    unit : `astropy.units.Unit`, optional
```

Comment:
Oh that was unintentional, I thought I got every `astropy.*`-link a `~`-prefix. I'll fix it.